test/lib/mayaUsd/render/vp2RenderDelegate/CMakeLists.txt

```diff
@@ -67,8 +67,6 @@ if (CMAKE_WANT_MATERIALX_BUILD)
             "MAYA_PLUG_IN_PATH=${CMAKE_INSTALL_PREFIX}/lib/maya"
             "LD_LIBRARY_PATH=${ADDITIONAL_LD_LIBRARY_PATH}"
             "MAYA_LIGHTAPI_VERSION=${MAYA_LIGHTAPI_VERSION}"
-            "PXR_MTLX_STDLIB_SEARCH_PATHS=${PXR_USD_LOCATION}/libraries"
-            "PXR_MTLX_STDLIB_SEARCH_PATHS=${CMAKE_INSTALL_PREFIX}/libraries"
```

Comment:
This test is inside of `CMAKE_WANT_MATERIALX_BUILD`, but the one below is not. Is that a mistake or does it need these variables regardless of whether or not we have MaterialX build?

Reply:
We don't need the variables but it does not hurt if they are there anyways. Probably means that we do not need to check for CMAKE_WANT_MATERIALX_BUILD in test.cmake, which means this PR is 100% correct and does not need modifications.